src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -55,9 +56,43 @@
   <UsingTask AssemblyFile="$(XPlatSourceBuildTasksAssembly)" TaskName="WritePackageUsageData" />
   <UsingTask AssemblyFile="$(XPlatSourceBuildTasksAssembly)" TaskName="WriteUsageReports" />
 
-  <!-- Update NuGet feeds in the repo -->
+  <Target Name="CopyNuGetConfig"
```

Comment:
Should this target have inputs & outputs for incremental support?

Reply:
Probably - will add inputs/outputs.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18478/commits/55d2441f88af313329b757886fc4d72cf7c177c8